### PR TITLE
[datapipe] Support wenet datapipe

### PR DESCRIPTION
**Pull request by mlxu995**

### CODE REVIEW

**Comment by a122760** [comment]
可以支持正负样本平衡的dataloader吗，当前框架下有什么好的实现方式呢？

**Comment by cdliang11** [comment]
Good Job!

**Comment by mlxu995** [comment]
- result of e2e loss on keyword "hi xiao wen" 
image
and on keyword "ni hao wen wen" 
image
image

- result of ctc loss on keyword "hi xiao wen" 
image
and on keyword "ni hao wen wen" 
image

File: wekws/bin/train.py [hunk 1]
```diff
@@ -108,11 +110,19 @@ def main():
     cv_conf['spec_aug'] = False
     cv_conf['shuffle'] = False
 
-    train_dataset = Dataset(args.train_data,
-                            train_conf,
-                            reverb_lmdb=args.reverb_lmdb,
-                            noise_lmdb=args.noise_lmdb)
-    cv_dataset = Dataset(args.cv_data, cv_conf)
+    # train_dataset = Dataset(args.train_data,
```
Comment (cdliang11) [inline wekws/bin/train.py:113]: 注释不需要了可以删掉

**Review by cdliang11 — APPROVED** [review]

**Comment by mlxu995** [comment]
> 可以支持正负样本平衡的dataloader吗，当前框架下有什么好的实现方式呢？就是每个iter都平衡正负样本。正负样本分开做两个list？

maybe this is helpful https://arxiv.org/pdf/1912.04486

**Comment by MM-0712** [comment]
@mlxu995
